inductiva/coastal/README.md

```diff
@@ -56,30 +82,31 @@ scenario = inductiva.coastal.CoastalArea(bathymetry=bathymetry,
                                          wave_period=5.5)
 ```
 
-Once the scenario is created, run the simulation as follows:
+Once the scenario is created, run the simulation by specifying the simulation
```

Comment:
I feel that this part is very explicit on the code. So you can just remove it. and at the end of script you pass to the next paragraph you have below with the visual.

Reply:
Better to be redundant. There were some complaints abt output_time_step definition, so I prefer to be thorough here

Reply:
But you say that below. For the simulate you can mention it afterwards. For the initialisation i I think it is very explicit.